instrumentation/servlet/servlet-3.0/library/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+dependencies {
+  library("javax.servlet:javax.servlet-api:3.0.1")
+
+  // FIXME: These dependencies need to be shadowed into the library.
+  library(project(":instrumentation:servlet:servlet-3.0:javaagent"))
```

Comment:
library instrumentations don't typically depend on javaagent instrumentation, could you move the helper into a common module or swap the dependency and have the javaagent depend on the library?

Reply:
Per Lauri's suggestion, I ended up copying over all the classes.